assets/src/dashboard/app/index.js

```diff
@@ -28,13 +28,20 @@ import { ThemeProvider } from 'styled-components';
  * Internal dependencies
  */
 import theme, { GlobalStyle } from '../theme';
+import { Route, RouterProvider } from './router';
+import { MyStoriesView, TemplatesGalleryView, MyBookmarksView } from './views';
+import { NavigationBar } from './components';
 
 function App() {
   return (
     <ThemeProvider theme={theme}>
-      <GlobalStyle />
-      <h1>{__('Dashboard', 'web-stories')}</h1>
-      <p>{__('Coming soon', 'web-stories')}</p>
+      <RouterProvider>
```

Comment:
To confirm: would merging this temporary disable the default WP stories list? Will we be able to test in the meantime?

Reply:
I don't think so because there is nothing on the dashboard other than a blank screen and these two strings.